programs/marginfi/src/instructions/marginfi_group/configure_bank.rs

```diff
@@ -71,16 +72,18 @@ pub fn lending_pool_setup_emissions(
     bank.emissions_rate = emissions_rate;
     bank.emissions_remaining = I80F48::from_num(total_emissions).into();
 
-    transfer(
+    transfer_checked(
```

Comment:
Any specific reason to have moved to `transfer_checked` some places and kept `transfer` in others?

Reply:
deprecated warning yelling at me

Reply:
lmao so you folded on this one
powerful linter